api/organisations/subscriptions/service.py

```diff
@@ -0,0 +1,27 @@
+from organisations.chargebee import (
```

Comment:
I'd say perhaps we should call this module `subscription_service` to be explicit?

Reply:
I did actually name it that, but with that, the import looks a little weird, no?

`from organisations.subscriptions.subscription_service import get_subscription_metadata`

Reply:
That looks fine to me, I find that happens a lot in python applications (duplicate names in module paths). I prefer it like that as it's more explicit.

Reply:
Yeah, I guess it;s not too bad